### DISCUSSION TITLE
Some questions about the Unikernel and Unikraft.

### DISCUSSION THREAD

**Discussion by hack3rcon** [unikraft/unikraft#542] [Q&A]
Hello,
Is Unikernel mean that my program running on a Hypervisor and without any OS? For example, Apache web server running without any OS.
The Unikraft is a platform to make Unikernel programs easier?

Thank you.

**Comment by marcrittinghaus** [comment]
Hi @hack3rcon,

yes and yes. Unikraft is a very thin layer that allows POSIX applications to directly run on the hypervisor. Usually, the application and Unikraft are built and linked together into a single executable file that can be booted for example on KVM. For NGINX, we have a working example.

Have a look at https://unikraft.org/docs/ for some help on getting started.

**Reply by hack3rcon** [reply]
Hi @marcrittinghaus 
Thank you so much.
How about Xen?

**Comment by razvand** [comment]
@hack3rcon, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
